models/models.js

```diff
@@ -35,9 +35,22 @@ const selectArticleID = (id) => {
     })
 }
 
-const selectArticles = () => {
+const selectArticles = (sort_by, order) => {
+  if(sort_by === undefined){
+    sort_by = 'created_at'
+  }
+  if(order === undefined){
+    order = 'DESC'
```

Comment:
Rather than doing this, you can tidy it up by just putting `sort_by = 'created_at'` and `order = 'DESC'` in the parameter to the model. 👍